packages/components/src/Badge/Badge.module.css

```diff
@@ -0,0 +1,101 @@
+.badge {
+  border-radius: var(--spacing-12);
+  color: var(--color-purple-800);
+  display: inline-block;
+  position: relative;
+  font-family: var(--typography-paragraph-extra-small-font-family);
+  font-size: var(--typography-paragraph-extra-small-font-size);
+  font-weight: var(--typography-paragraph-bold-font-weight);
+  letter-spacing: var(--typography-paragraph-extra-small-letter-spacing);
+  line-height: var(--typography-paragraph-extra-small-line-height);
+  padding: 1px var(--spacing-6);
+  min-width: 8px;
+  text-align: center;
+  background-color: var(--badge-background-color, var(--color-gray-300));
+
+  .reversed {
```

Comment:
Similar to what we were talking about in Avatar. Nesting the class like this will target the children of the element with the badge class:
![Screenshot 2025-01-09 at 2 02 53 PM](https://github.com/user-attachments/assets/2b4b8daf-db06-462c-b9d0-d0fa15e6fd62)

Whereas the original SCSS reversed class is at the root the badge stylesheet:
![Screenshot 2025-01-09 at 2 03 53 PM](https://github.com/user-attachments/assets/d46eb862-0310-470e-ac0d-e8e9aabfc565).

A handy thing to lean on here is hovering over the class name in VSCode within the stylesheet, which will bring up the prompt that shows how that class will target elements on the page

Reply:
We could use the `&` when nesting in CSS like this, which will increase the specificity of the styles.

![Screenshot 2025-01-09 at 2 11 26 PM](https://github.com/user-attachments/assets/2ba64cf9-e14e-4f05-aed3-dc197b91ea49)

However where possible we probably want to leave this un-nested so its specificity is relatively low so folks can override the styles if necessary